src/main/java/ch/uzh/ifi/hase/soprafs21/entity/User.java

```diff
@@ -79,6 +82,10 @@
     @OneToMany(mappedBy = "owner", orphanRemoval = true)
     private List<Dog> listOfDogs;
 
+    @JsonIgnore
```

Comment:
This is not what i meant. You cannot use @jsonIgnore in the entity only in DTO. But since you don't have owner in dto(I havent noticed that before) you do not need it there as well.